Riot/Modules/Analytics/SentryMonitoringClient.swift

```diff
@@ -28,9 +28,25 @@ struct SentryMonitoringClient {
             options.sampleRate = 0.1
             options.tracesSampleRate = 0.1
             
-            // Disable unnecessary network tracking
+            // Sentry swizzling shows up quite often as the heaviest stack trace when profiling
+            // We don't need any of the features it powers (see docs)
+            options.enableSwizzling = false
+            
+            // WatchdogTermination is currently the top issue but we've had zero complaints
+            // so it might very well just all be false positives
+            options.enableWatchdogTerminationTracking = false
+            
+            // Disabled as it seems to report a lot of false positives
+            options.enableAppHangTracking = false
+            
+            // Most of the network requests are made Rust side, this is useless
```

Comment:
Probably should have the original comment given we're not using Rust here 😅

```suggestion
            // Disable unnecessary network tracking
```